.github/workflows/lint.yml

```diff
@@ -40,9 +40,12 @@ jobs:
         env:
           HOME: /root
         run:
-          "\n          opam install -y ocamlformat.0.26.1\n          git config --global
-          --add safe.directory \"$GITHUB_WORKSPACE\"\n          opam exec -- pre-commit
-          run --verbose --all lint-ocaml\n        "
+          "\n          # When installing ocamlformat.0.26.1 OPAM will try to rebuild
```

Comment:
I was following this PR from another PR.

Is this portion to be quoted and formatted like this (granted it already looked weird before this PR)?

Reply:
I agree it's a bit gross/confusing -- it's because this file is being generated by `jsonnet` (see `lint.jsonnet` above). In an ideal world the tool would know to use pipe syntax for multi-line yaml strings. (cc @aryx)